asyncio/event.py

```diff
@@ -33,15 +46,23 @@ def set(self):
         self.state = True
 
     def clear(self):
+        """Clear the event."""
+
         self.state = False
 
     async def wait(self):
+        """Wait for the event to be set. If the event is already set then it returns
+        immediately.
+
+        This is a coroutine.
+        """
+
         if not self.state:
             # Event not set, put the calling task on the event's waiting queue
             self.waiting.push_head(core.cur_task)
             # Set calling task's data to the event's queue so it can be removed if needed
             core.cur_task.data = self.waiting
-            yield
+            await core.sleep(0)
```

Comment:
I am still wrapping my head around this library and the await / yield keywords and things. This change may be normal and expected, but just wanted to note it is the only thing I've come across so far that isn't docstrings.